client/src/components/App/AppToolbar/TopToolbar/TopToolbarStyles.ts

```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from '@material-ui/styles'
+
+const useStyles = makeStyles({
+    toolbar: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+
+    },
+    rightToolbarSection: {
```

Comment:
I think more meaningful name will help to understand what is it.

Reply:
זה השם החדש שהוצע

Reply:
פשוט לי זה לא כ"כ היה ברור מה זה סקשן ימני, מה הוא כולל אבל אם זה מה שהוצע סבבה